### PR TITLE
Company Card Component with Hover Detail Reveal

### PR DESCRIPTION
**Pull request by SmileyFaceZ**
This pull request introduces a new Company Card component with the following updates:

* Basic Company Card Component
* Hover Effect for Detail Reveal

### CODE REVIEW

**Comment by xNatthapol** [comment]
Thank you for the Company Card component. I tested it, and it works good, but I'll wait for another review.

**Review by Peanutt47 — APPROVED** [review]
LGTM

**Review by xNatthapol — APPROVED** [review]
Thank you for the Company Card component. I tested it, and it works good, but I'll wait for another review.

**Pull request merged**
